### PR TITLE
Initialize

### PR DESCRIPTION
**Pull request by ryonryon**
Can you review the changes?

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
+// compiled file dir
```
Comment (axross) [inline .gitignore]: Use `#` instead

File: app.yaml [hunk 2]
```diff
@@ -0,0 +1 @@
+runtime: nodejs8
```
Comment (axross) [inline app.yaml]: Prefer to use Node v10 instead

File: package.json [hunk 3]
```diff
@@ -0,0 +1,33 @@
+{
+  "name": "daddy-backend",
+  "version": "1.0.0",
```
Comment (axross) [inline package.json]: Set version `0.0.0`

File: package.json [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "dev": "micro-dev",
+    "start": "micro"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/agnosticful/daddy-backend.git"
+  },
+  "author": "",
+  "license": "MIT",
```
Comment (axross) [inline package.json]: Do not put any license

**Review by axross — DISMISSED** [review]
LGTM

Request you to put server code with micro in the next pull request.

**Comment by ryonryon** [comment]
@axross Can you check again? I modified the result of the review.

File: package.json [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "dev": "micro-dev",
+    "start": "micro"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/agnosticful/daddy-backend.git"
+  },
+  "author": "",
+  "license": "",
```
Comment (axross) [inline package.json]: Remove `license` field itself

**Review by axross — CHANGES_REQUESTED** [review]
🐧

File: package.json (hunk 5 above)
Comment (ryonryon) [reply package.json]: @axross Removed license field.

File: package.json [hunk 6]
```diff
@@ -14,20 +14,26 @@
...
     "ts-node": "^8.0.2",
     "typescript": "^3.3.3"
   },
   "dependencies": {
     "micro": "^9.3.3",
-    "microrouter": "^3.1.3"
+    "apollo-server-micro": "^2.4.2",
+    "eslint": "^5.14.1",
+    "graphql": "^14.1.1",
+    "graphql-tools": "^4.0.4"
```
Comment (axross) [inline package.json]: There's several dependencies added than before. Is this a mistake?

**Comment by ryonryon** [comment]
@axross Can you check again? I modified the result of the review.

File: package.json [hunk 7]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "daddy-backend",
+  "version": "0.0.0",
+  "description": "",
+  "main": "built/main.js",
```
Comment (axross) [inline package.json]:
```suggestion

```

File: package.json [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+  },
+  "author": "",
+  "bugs": {
+    "url": "https://github.com/agnosticful/daddy-backend/issues"
+  },
+  "homepage": "https://github.com/agnosticful/daddy-backend#readme",
+  "devDependencies": {
+  },
+  "dependencies": {
+  }
```
Comment (axross) [inline package.json]: Delete empty `"devDependencies"` and `"dependencies"` sections.

**Review by axross — APPROVED** [review]
LGTM

**Pull request merged**